en-US/fx_android/whatsnew/android_62.lang

```diff
@@ -0,0 +1,6 @@
+## NOTE: These strings are displayed on Google Play in the What’s new section for Firefox.
+## NOTE: See https://l10n.mozilla-community.org/stores_l10n/locale/fx_android/beta/
+
+
+;We're working to make Firefox for Android better with each new release. Read the release notes to learn about any new features, bug fixes, and performance improvements at https://www.mozilla.org/en-US/firefox/android/notes/
```

Comment:
This doesn't match the string in https://github.com/mozilla-l10n/stores_l10n/pull/166, which uses proper apostrophes.